src/s1reader/s1_info.py

```diff
@@ -48,27 +56,109 @@ def _plot_bursts(safe_path, output_dir="burst_maps"):
+
+    # Check that they have all the necessary kmls
+    if not overlay_path.exists():
+        raise ValueError(f"{overlay_path} does not exist.")
+    root = ET.parse(overlay_path).getroot()
+
+    # point_str looks like:
+    # <coordinates>-102.552971,31.482372 -105.191353,31.887299...
+    point_str = list(elem.text for elem in root.iter("coordinates"))[0]
+    coords = [p.split(",") for p in point_str.split()]
+    lons, lats = zip(*[(float(lon), float(lat)) for lon, lat in coords])
+    if temp_dir is not None:
+        shutil.rmtree(temp_dir)
+    return [min(lons), min(lats), max(lons), max(lats)]
+
+
+def _bounds_from_bursts(safe_path: Union[Path, str]) -> List[float]:
+    """Get the bounding box of the frame from the union of all burst bounds."""
+    # Get all the bursts from subswath 1, 2, 3
+    bursts = get_bursts(safe_path)
+    # Convert the border (list of polygons) into a MultiPolygon
+    all_borders = [shapely.geometry.MultiPolygon(b.border) for b in bursts]
+    # Perform a union to get one shape for the entire frame
+    border_geom = shapely.ops.unary_union(all_borders)
+    # grab the bounds and pad as needed
+    return list(border_geom.bounds)
+
+
 EXAMPLE = """
 Example usage:
 
     # Print all bursts in a Sentinel-1 SLC product
-    s1_info.py S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.zip
+    s1_info S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.zip
 
     # Print only the burst IDs
-    s1_info.py S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.SAFE --burst-id
+    s1_info S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.SAFE --burst-id
 
     # Print burst ids for all files matching the pattern
-    s1_info.py -b S1A_IW_SLC__1SDV_2018*
+    s1_info -b S1A_IW_SLC__1SDV_2018*
 
     # Print only from subswath IW1, and "vv" polarization
-    s1_info.py -b S1A_IW_SLC__1SDV_2018* --iw 1 --pol vv
+    s1_info -b S1A_IW_SLC__1SDV_2018* --iw 1 --pol vv
 
     # Get info for all products in the 'data/' directory
-    s1_info.py data/
- 
+    s1_info data/
+
+    # Print the bounding box of each frame
+    s1_info --bbox data/
+
+    # Using https://github.com/scottstanie/sardem , create a DEM covering the SLC product
+    s1_info --bbox S1A_IW_SLC__1SDV_20220226T124745_20220226T124812_042084_050378_F69A.zip |
+        cut -d':' -f2 |     # separate the bbox from the label
+        tr -d ',[]'   |     # remove brackets and commas
+        xargs sardem --data cop --bbox     # pass the bbox to sardem as an argument
+
+
     # Plot the burst map, saving files into the 'burst_maps/' directory
-    s1_info.py S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.SAFE/ --plot
-    s1_info.py S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.zip -p -o my_burst_maps
+    s1_info S1A_IW_SLC__1SDV_20180601T000000_20180601T000025_021873_025F3D_9E9E.SAFE/ --plot
```

Comment:
This option does not recognize s1reader.utils and the plot_bursts module. shouldn't it be added to setup?

Reply:
Oh I see, maybe it's only working for me since I installed it with `pip install -e .`. 

I'll add __init__.py to the utils folder and it should work